com.unity.render-pipelines.high-definition/Editor/Compositor/CompositorWindow.cs

```diff
@@ -12,10 +12,11 @@ namespace UnityEditor.Rendering.HighDefinition.Compositor
 {
     internal class CompositorWindow : EditorWindow
     {
-        static partial class Styles
+        static class Styles
         {
-            static public readonly GUIContent k_EnableCompositor = EditorGUIUtility.TrTextContent("Enable Compositor", "Enabled the compositor and creates a default composition profile.");
-            static public readonly GUIContent k_RemoveCompositor = EditorGUIUtility.TrTextContent("Remove compositor from scene", "Removes the compositor and any composition settings from the scene.");
+            static public GUIContent windowTitle { get; } = new GUIContent("Graphics Compositor");
+            static public GUIContent enableCompositor { get; } = EditorGUIUtility.TrTextContent("Enable Compositor", "Enabled the compositor and creates a default composition profile.");
```

Comment:
@RSlysz IIRC the convention for static should be k_RemoveCompositor  ? Or have it change?

Reply:
I've changed them to be properties instead of fields, @RSlysz do we have a naming convention for static properties?

Reply:
Convention is AFAIK:
- static Type s_Name
- static readonly Type k_Name
- const Type k_Name
- public <any visibility> Type name
And we try to not expose static field or const field but prefer expose them with a property in the Unity code base.
So this seams good.

Reply:
Complete guidelines here: https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs